app/views/components/_no_sam_verification_header.html.erb

```diff
@@ -10,3 +11,8 @@
     <p>*NOTE: We periodically check if all users are listed in SAM.gov. It may be possible that you registered after the most recent check. However, we will always run a check at least once per day, so you may have to wait up to 24 hours between registration and bidding.</p>
   </div>
 </div>
+
+<div class='usa-alert usa-alert-warning warning-show-trigger' style='display:none'>
```

Comment:
I am going to look into adding ARIA metadata on these as well, but that is also something that could be deferred for a future PR